includes/widgets/video.php

```diff
@@ -1174,7 +1174,7 @@ private function render_hosted_video() {
 
 		$video_params = $this->get_hosted_params();
 		?>
-		<video class="elementor-video" src="<?php echo esc_url( $video_url ); ?>" <?php Utils::print_html_attributes( $video_params ); ?>></video>
+		<video class="elementor-video" src="<?php echo esc_attr( $video_url ); ?>" <?php Utils::print_html_attributes( $video_params ); ?>></video>
```

Comment:
Why?

Reply:
Because it's not always a URL, when in preview mode - it's a base64 string.

Reply:
Please add a comment in the code